spec/lib/twingly/http_spec.rb

```diff
@@ -505,6 +505,31 @@ class CustomError < StandardError; end
       end
     end
 
+    describe "headers" do
+      let(:headers) do
+        {
+          "Content-Type" => "application/json",
+        }
+      end
+
+      let(:request_response) do
+        client.get(url, headers: headers)
+      end
+
+      before do
+        headers_in_body_lamba = lambda do |request|
+          { body: request.headers.to_json }
+        end
+
+        stub_request(:get, url)
+          .to_return(&headers_in_body_lamba)
+      end
+
+      it "does request with specified headers" do
+        expect(JSON.parse(response.fetch(:body))).to include(headers)
```

Comment:
Not a new problem, but I just happened to notice it now. We should require `json` somewhere since we use both `JSON` and `#to_json` in a few places.

Reply:
Nice 🐟 , faf0c3b